sharding-jdbc-example/broadcast-table-example/pom.xml

```diff
@@ -25,6 +25,7 @@
         <module>broadcast-table-raw-jdbc-example</module>
         <module>broadcast-table-spring-boot-example</module>
         <module>broadcast-table-spring-namespace-example</module>
+        <module>broadcast-table-raw-jdbc-multi-example</module>
```

Comment:
it's better to move the code to `broadcast-table-raw-jdbc-example` directly, and delete `t_coutry` relevant code, Sportman contains feature of Country.

Reply:
Ok,I will refactor that.